atc/api/pipelineserver/order.go

```diff
@@ -32,10 +31,10 @@ func (s *Server) OrderPipelines(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = team.OrderPipelines(pipelinesRefs)
+	err = team.OrderPipelines(pipelinesNames)
 	if err != nil {
 		logger.Error("failed-to-order-pipelines", err, lager.Data{
-			"pipeline-refs": pipelinesRefs,
+			"pipeline_names": pipelinesNames,
 		})
 		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
I tried to order a pipeline that doesn't exist, and it gave a very unhelpful message (note the empty Body)
```
failed to order pipelines: Unexpected Response
Status: 500 Internal Server Error
Body:

```

It would be nice to get either a 404 or some sort of indication that i mistyped a pipeline name

Reply:
Updated to return a 400 error (not 404, which is intended to mean "The server has not found anything matching the Request-URI" - but this isn't a Request URI issue, it's in the request body). There's context in the response body, at least